app/profile/index.js

```diff
@@ -14,6 +14,7 @@ export default class LoginScreen extends React.Component<{||}> {
   };
 
   onLogout = () => {
+    // TODO: clean the Relay offline cache (?)
```

Comment:
Would not this be: 
`AsyncStore.removeItem('@OfflineStore:key');`?

Reply:
Yes. But I don't want to implement it now since this part of the application is not ready yet. We will have to wrap the `AsyncStore` and clean the cache here. Let's ignore this `profile` part for now (it's only for us).